starters/express-apollo-prisma/src/graphql/mappers/technology.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { mapTechnology } from './technology';
+import { TechnologyEntity } from '@prisma/client';
+import { Technology } from '../schema/generated/types';
+
+describe('.mapTechnology', () => {
+	describe('when called', () => {
+		it('returns expected result', () => {
```

Comment:
How to use `describe` and `it` is always a matter of taste. Please feel free to make it whatever you like.

I see little value in having the intermediate `describe` ("when called") and would personally drop it. In the same fashion I would rename "returns expected result" to "creates a Technology from a TechnologyEntity". To me, it is implied that ".mapTechnology is called" and "it returns the expected result".